src/derivative_wrappers.jl

```diff
@@ -2,10 +2,13 @@ struct SensitivityAlg{CS,AD,FDT} <: DiffEqBase.DEAlgorithm
   autojacvec::Bool
   quad::Bool
   backsolve::Bool
+  checkpointing::Bool
 end
-Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central},autojacvec=autodiff,quad=true,backsolve=false)
+Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central},
```

Comment:
We should default to interpolating if `sol.dense`, otherwise default to checkpointing.